app/src/main/java/lt/vilnius/tvarkau/fragments/BaseReportListFragment.kt

```diff
@@ -52,16 +53,26 @@ abstract class BaseReportListFragment : BaseFragment(), ReportListView {
                 .subscribeOn(ioScheduler)
                 .observeOn(uiScheduler)
                 .subscribe({
-                    reloadData()
+                    reloadReports = true
```

Comment:
This won't work. If unsubscribtion is done in `onDestroyView` ReportsList fragment won't receive an event of applied filters from ReportsFilterFragment.  
We need to keep subscribed `RxBus` after `onDestroyView`.